### PR TITLE
Adding json and xml formats response for project show

### PR DESCRIPTION
**Pull request by carolyncole**
refs #287

WIP

### CODE REVIEW

**Review by bess — COMMENTED** [review]
Looks great!

**Comment by hectorcorrea** [comment]
Nice, I like the XML payload idea to allow us to export what we send to Mediaflux.

**Review by hectorcorrea — APPROVED** [review]

**Review by bess — APPROVED** [review]

File: app/models/project_mediaflux.rb [hunk 1]
```diff
@@ -43,6 +44,16 @@ def self.project_values(project:)
     values
   end
 
+  def self.xml_payload(project:, xml_namespace: nil)
```
Comment (hectorcorrea) [inline app/models/project_mediaflux.rb:47]: @carolyncole In looking at this, I think this is returning the XML to create the namespace only, but the original code (https://github.com/pulibrary/tiger-data-app/blob/main/app/models/project_mediaflux.rb#L3-L21) sends two requests to Mediaflux: One to create the namespace and one to create the collection assets underneath. I am not sure how we want to represent this in the what will produce for people to reuse.

Comment (carolyncole) [reply app/models/project_mediaflux.rb:47]: @hectorcorrea correct.  It only creates the collection asset.create XML, not the namespace create XML.

Comment (carolyncole) [reply app/models/project_mediaflux.rb:47]: We could possible attach 3 files to the email, one for the collection, one for the namespace, and then the json representation of the data.

**Pull request merged**